pkg/queue/sharedmain/main.go

```diff
@@ -410,11 +413,17 @@ func buildServer(ctx context.Context, env config, transport http.RoundTripper, p
 		composedHandler = requestLogHandler(logger, composedHandler, env)
 	}
 
+	var port string
 	if enableTLS {
-		return pkgnet.NewServer(":"+env.QueueServingTLSPort, composedHandler), drainer
-	}
-
-	return pkgnet.NewServer(":"+env.QueueServingPort, composedHandler), drainer
+		mainService.name = `mainTls`
```

Comment:
Nit: ` -> "

Reply:
I did not get that :)

Reply:
Was a bit cryptic, sorry. We don't need the multiline-string syntax here, right?